06_create_cluster.sh

```diff
@@ -49,4 +49,22 @@ if [[ -n "${APPLY_EXTRA_WORKERS}" ]]; then
     fi
 fi
 
+if [[ "${ENABLE_METALLB}" == "true" ]]; then
+	if [[ ! -z "${$METALLB_MODE}" ]]; then
+		if [[ ${METALLB_MODE} == "l2" ]]; then
+			pushd metallb
+			make config_l2
```

Comment:
From the integration point of view with dev-scripts, this sounds a little bit unusual - ie using directly the related scripts seems also easier here

Reply:
changed.